doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -68,30 +94,159 @@ between tenants
+### 5.3.1 Node hardening: Securing Kubernetes hosts         
+When an operating system or application is installed, it comes with default
+settings. Usually, all ports are open, and all application services are turned
+on. In other words, freshly installed assets are highly insecure.
 
-## 5.4 Authentication & Authorisation
+Ensure Kubernetes nodes are secure, hardened and configured correctly following
+well known security frameworks. Security benchmarks, for example, CIS benchmarks
+are a set of configuration standards and best practices designed to help ‘harden’
+the security of their digital assets. 
+
+### 5.3.2 Restrict direct access to nodes
+Restrict root/administrative access to Kubernetes nodes while avoiding direct
+access to nodes for operational activities including debugging, troubleshooting,
+and other tasks.
+
+### 5.3.3 Vulnerability assessment
+Vulnerability assessments are a crucial part of IT risk management lifecycles.
+The mitigation of the vulnerabilities helps in protecting systems and data from unauthorized access and breaches.
+Implement necessary vulnerability scanner tools (Ex: OpenVAS and many other
+opensource and commercial tools) to identify threats and flaws within the
+infrastructure that represents potential risks.
+
+### 5.3.4 Patch management 
+Patch management is another key aspect of IT risk management lifecycle for
+security, compliance, uptime, feature enhancements and so on. Implement
+necessary patch management to ensure your environment is not susceptible to
+exploitation.
+    
+## 5.4 Securing Kubernetes orchestrator 
+
+### 5.4.1 Control network access to sensitive ports
+Kubernetes clusters usually listen on a range of well-defined and distinctive
+ports which makes it easy to identify the clusters and attack them. Hence, it is
+highly recommended to configure authentication and authorization on the cluster
+and cluster nodes.
+
+Here is an overview of the default ports used in Kubernetes. Make sure that your
+network blocks access to ports and consider limiting access to the Kubernetes
+API server except from trusted networks.
+
+**Master node(s):**
+
+| Protocol | Port Range | Purpose                 |
+| -------- | ---------- | ----------------------- |
+| TCP      | 6443-      | Kubernetes API Server   |
+| TCP      | 2379-2380  | etcd server client API  |
+| TCP      | 10250      | Kubelet API             |
+| TCP      | 10251      | kube-scheduler          |
+| TCP      | 10252      | kube-controller-manager |
+| TCP      | 10255      | Read-Only Kubelet API   |
+
+**Worker nodes:**
+
+| Protocol | Port Range  | Purpose               |
+| -------- | ----------- | --------------------- |
+| TCP      | 10250       | Kubelet API           |
+| TCP      | 10255       | Read-Only Kubelet API |
+| TCP      | 30000-32767 | NodePort Services     |
+
+### 5.4.2 Controlling access to the Kubernetes API
+The Kubernetes platform is controlled using API requests and as such is the
```

Comment:
Does the "is" in "... and as such is ..." refers to the platform or APIs? Neither by themselves is "the first line of defense" but more like a target for attackers. The "." is missing after "concern".

Reply:
Kubernetes platform is API driven, and these APIs are the first items to be secured in order to defend against the attackers.

Reply:
@ragashreeshekar Sorry, I should have been clearer. The sentence is confusing and it is not clear if the "first line of defense" is referring to the platform or the APs?  Also, I think the APis are the first to be attacked and we need mechanisms to defend against that.